pkg/azure/keyvault/store.go

```diff
@@ -65,7 +72,22 @@ func (s *Store) Resolve(ctx context.Context, keyName string, keyValue string) (s
 	if err := s.Connect(ctx); err != nil {
 		return "", err
 	}
-
+	// Check if the keyValue is set to a full ID or just the secret name. The keyValue is only considered
+	// an ID if all 3 pieces of info (keyvault URL, secret name, and version) parse, otherwise the configured vault URL is
```

Comment:
This comment implies that all 3 parts are required but I believe parse still works with a missing version (which is great).

Reply:
Whoops yeah this is the behavior I originally had but changed it and missed the comment